tests/circuit_graph/test_qasm.py

```diff
@@ -724,23 +759,23 @@ def qnode(x):
 
         # operations
         assert gates[0].name == "h"
-        assert gates[0].wires == Wires([0])
+        assert gates[0].wires == [0]
 
         assert gates[1].name == "ry"
-        assert gates[1].wires == Wires([0])
-        assert gates[1].data == [params[1]]
+        assert gates[1].wires == [0]
+        assert gates[1].params == [params[1]]

```

Comment:
This got changed to `data` after #719, however `gates` is a list of _qiskit_ gates, not PennyLane operations, and qiskit uses `params`!

Reply:
These tests only run if Qiskit is installed, hence why it was not caught by the GitHub CI

Reply:
Whoops!